src/cli/one_helper/onezone_helper.rb

```diff
@@ -83,25 +83,30 @@ def format_pool(options)
         table
     end
 
-    def set_zone(zone_id)
-      zone = factory(zone_id)
-      rc = zone.info
+    def set_zone(zone_id, overwrite_file = true)
```

Comment:
Prefer not to use options by default. Check that set_zone is not being used anywhere (this file should be only used in onezone command)